thoth/user_api/api_v1.py

```diff
@@ -43,7 +43,10 @@
 from thoth.python import Project
 from thoth.python.exceptions import ThothPythonException
 from thoth.user_api.payload_filter import PayloadProcess
-from thoth.user_api.openapi_server import metrics
+from thoth.user_api.openapi_server import _METRIC_CACHE_HIT_ADVISER_AUTHENTICATED
```

Comment:
The module from which the import is done is still the same. Will this address the issue?

Reply:
correct, then prometheus_flask_exporter expects metrics to be present all in openapi_server, we cannot import it and use it in a method, we should do something like

the best solution using `prometheus_flask_exporter` seems:

@app.route('/advise/python')
@metrics.counter(
    'thoth_user_api_cache_hit_rate',
    'Thoth User API cache hit rate',
    labels={
        'is_authenticated': lambda: response['authenticated'],
        'service': "adviser",
        `env`: Configuration.THOTH_DEPLOYMENT_NAME
    })
def advise_python(inputs):
    return advise_python

but it does not seem the right way.

Reply:
also `from .openapi_server import metrics` failed @fridex :/

Reply:
maybe I have another idea

Reply:
Using after_request, wdyt @fridex ?